tests/cli/test/src/integration.test.js

```diff
@@ -8,7 +8,8 @@ const {
   getNetwork,
   truffleExec,
   run,
-  copy
+  copy,
+  setMockStdlibVersion
```

Comment:
As we are going to do a bit of tech debt this sprint (are we? :stuck_out_tongue:), we may want to replace all the `stdlib` to `dependency` or `package` as appropiate.

Reply:
Hmm I'd prefer to tackle other more pressing tech debt items than renaming stdlib nomenclature in tests.